src/common/mfu_daos.c

```diff
@@ -585,7 +680,7 @@ static int daos_connect(
 }
 
 /* Mount DAOS dfs */
-static int daos_mount(
+static int mfu_dfs_mount(
```

Comment:
@daltonbohning what determines if we use the mfu_dfs_ prefix vs. the mfu_daos_ prefix?

Reply:
@daltonbohning never mind, I went back and re-read our messages about this, so I see this has to do with whether or not the function returns positive or negative error codes.

Reply:
@dsikich My thinking here was to use `mfu_dfs` since this is a wrapper function specifically for the DFS API.
Right, the idea is that we can know what sort of error code to expect.
Although... this function doesn't return the errno code from DFS. It generically returns -1 on error.
I'm on the fence about using `mfu_dfs` vs `mfu_daos` here.
What do you think?

Reply:
@daltonbohning I guess the more I think about it I like using dfs_ in the name for functions that use that API, since we have a lot of functionality now that doesn't use DFS.

Reply:
@dsikich Yeah, and we could always change this later if needed. This function is `static`, so it's easy enough to rename it